integration/docker/README.md

```diff
@@ -24,3 +26,38 @@ docker run -e ALLUXIO_MASTER_HOSTNAME=ec2-203-0-113-25.compute-1.amazonaws.com a
 Additional configuration files can be included when building the image by adding them to the
 `integration/docker/conf/` directory. All contents of this directory will be
 copied to `/opt/alluxio/conf`.
+
+
+## Building docker image with fuse support
+To build a docker image with fuse support, run
```

Comment:
Can you also add a link to the FUSE doc page?

Reply:
Done.